docs/en/quickstart/sdk/python_sdk.md

```diff
@@ -133,98 +133,135 @@ connection = engine.connect()
 
 ### Create Database
 
-Use the `connection.execute()` interface to create database `db1`:
+Use the `connection.exec_driver_sql()` interface to create database `db1`:
 
 ```python
 try:
-    connection.execute("CREATE DATABASE db1")
+    connection.exec_driver_sql("CREATE DATABASE db1")
```

Comment:
will `execute` still available from SQLAlchemy 2 ?

Reply:
Usable，SQLAlchemy 2.0 changes how execute works compared to 1.4. The recommendation is to use ORM Query whenever possible. For Core SQL, use exec_driver_sql, as execute doesn't support DDL. There are also differences in how DML and DQL statements are handled.